content/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -11,10 +11,12 @@ You can set variables specifically for each workspace or you can create variable
 
 ~> **Note:** Variable sets are in beta.
 
+### Permissions
 You must have [`read variables` permission](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in your organization.
 
 To create new variable sets and apply them to workspaces, you must be part of a team with [manage workspaces](/cloud-docs/users-teams-organizations/permissions#organization-owners) permissions. To create and edit workspace-specific variables within a workspace, you must have [read and write variables](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) for that workspace.
 
+### Create and Edit Variables
```

Comment:
Let's leave this as part of the introduction (it's just meant to introduce the options for managing variables). And lets move it to be above the variables section, after the intro that's already there. And I think we can just say "You can create and edit variable sets through...." and remove the permissions statement as it's now redundant and we have a beautiful new flagged section.